tools/smoove_lumpy.cwl

```diff
@@ -66,38 +70,21 @@ arguments:
       tabix $(inputs.output_basename)-smoove.genotyped.vcf.gz
 
 inputs:
-  reference: { type: File,  secondaryFiles: [.fai], label: Fasta genome assembly with index }
-  input_tumor_align:
-    type: File?
-    secondaryFiles: |
-      ${
-        var path = inputs.input_tumor_align.location+".crai";
-        if (inputs.input_tumor_align.nameext == ".bam"){
-          path = inputs.input_tumor_align.location+".bai";
-        }
-        return { "location": path, "class": "File"};
-      }
-
-  input_normal_align:
-      type: File?
-      secondaryFiles: |
-        ${
-          var path = inputs.input_normal_align.location+".crai";
-          if (inputs.input_normal_align.nameext == ".bam"){
-            path = inputs.input_normal_align.location+".bai";
-          }
-          return { "location": path, "class": "File"};
-        }
+  reference: { type: 'File',  secondaryFiles: [{pattern: '.fai', required: true}], doc: "Fasta genome assembly with samtools index" }
+  input_tumor_align: { type: 'File?', secondaryFiles: [{pattern: '.crai', required: false}, {pattern: '.bai', required: false}], doc: "BAM/CRAM file containing aligned reads from the tumor sample." }
+  input_normal_align: { type: 'File?', secondaryFiles: [{pattern: '.crai', required: false}, {pattern: '.bai', required: false}], doc: "BAM/CRAM file containing aligned reads from the normal sample." }
+  exclude_bed: {type: 'File?', doc: "Bed file with regions to exlude from analysis, i.e. non-canonical chromosomes. Highly recommneded."}
   duphold_flag: {type: ['null', {type: enum, name: duphold_flag, symbols: ["Y", "N"] }], doc: "Run Brent P duphold and annotate DUP and DEL with depth change", default: "Y"}
   disable_smoove: {type: ['null', {type: enum, name: disable_smoove, symbols: ["Y", "N"] }], doc: "Disable smoove filtering, just do standard lumpy_filter", default: "N"}
-  exclude_bed: {type: ['null', File], doc: "Bed file with regions to exlude from analysis, i.e. non-canonical chromosomes. Highly recommneded."}
```

Comment:
Did this input get removed? OR am I missing it somewhere?

Reply:
No it got moved up to line 76. I think I just grouped all the files together when I was cleaning it up.